pkg/apis/build/v1alpha1/builder_validation.go

```diff
@@ -0,0 +1,31 @@
+package v1alpha1
+
+import (
+	"context"
+
+	"knative.dev/pkg/apis"
+)
+
+func (b *Builder) SetDefaults(ctx context.Context) {
+	if b.Spec.UpdatePolicy == "" {
+		b.Spec.UpdatePolicy = Polling
```

Comment:
This brings me so much joy.

Reply:
There is probably one other validation I would like to do here, is verify that the builder image provided is a valid image reference.